core/os/input_event.cpp

```diff
@@ -1000,6 +1000,23 @@ String InputEventScreenDrag::as_text() const {
 	return "InputEventScreenDrag : index=" + itos(index) + ", position=(" + String(get_position()) + "), relative=(" + String(get_relative()) + "), speed=(" + String(get_speed()) + ")";
 }
 
+bool InputEventScreenDrag::accumulate(const Ref<InputEvent> &p_event) {
+
+	Ref<InputEventScreenDrag> drag = p_event;
+	if (drag.is_null())
+		return false;
```

Comment:
Brackets are missing here.

Reply:
Here I just imitated `InputEventMouseMove::accumulate()`. Since in Godot 3 we are not enforcing the always-use-braces style, I decided to just do the same as in that other function, for consistency.